webpack/webpack.prod.config.js

```diff
@@ -86,9 +92,20 @@ module.exports = {
   optimization: {
     minimize: Boolean(process.env.WEBPACK_DEPLOY),
     minimizer: [
-      new UglifyJsPlugin({
-        exclude: /codemirror/,
-        parallel: true
+      new TerserPlugin({
```

Comment:
Changing horses here, huh? Why?

Reply:
this is the default in webpack now. the big change is uglify only works on es5, it's a little buggy, anda little slower. https://gist.github.com/mattkrick/6426ed85351b333b6408dbb7858b81c5

Reply:
Cool!